azureLogIngestion/adapter/index.test.ts

```diff
@@ -25,10 +26,11 @@ describe("Adapter", () => {
         expect(adapter.eventProcessor.batch.getBatchSize()).toEqual(0)
         expect(adapter.eventProcessor.client).toBeInstanceOf(telemetry.events.EventClient)
 
-        expect(adapter.spanProcessor).toBeInstanceOf(SpanProcessor)
-        expect(adapter.spanProcessor.batch).toBeInstanceOf(telemetry.spans.SpanBatch)
-        expect(adapter.spanProcessor.batch.getBatchSize()).toEqual(0)
-        expect(adapter.spanProcessor.client).toBeInstanceOf(telemetry.spans.SpanClient)
+        // expect(adapter.spanProcessor).toBeInstanceOf(SpanProcessor)
```

Comment:
Why are we commenting these tests out? Don't we still want to test this functionality?

Reply:
@kolanos I'm enabling/testing the otel equivalent now; they'll be up in the next commit. Or did we want to keep the previous DT version of the span processor?

Reply:
I think we want to keep both. If you want to move the `opentelemetry` module into the same directory `adapter` is in to avoid confusion, I'm +1 there. I see `OpenTelemetryAdapter` a s a separate adapter that can be run in parallel with the telemetry SDK one.

Reply:
> So I ran `npm` against nodejs 12.x, are you doing the same? Wondering why the `package-lock.json` is so different. Does this diff happen when you:
> 
> ```
> git checkout otel package-lock.json
> npm install
> ```
I'm running node 12.19, though I think I've inherited a v1 package.json from an earlier install. (Easy enough to fix with a reinstall.)